packages/block-library/src/post-excerpt/block.json

```diff
@@ -1,4 +1,17 @@
 {
 	"name": "core/post-excerpt",
-	"category": "layout"
+	"category": "layout",
+	"attributes": {
+		"excerptLength": {
```

Comment:
why not just "lenght"?

Reply:
It's clearer that it refers to the excerpt part. The block also has the read more text and could evolve to have other things in the future.

Reply:
This should probably be `wordCount` or `wordLength`.

Reply:
wordCount sounds like the most accurate to me. Let's change it.

Reply:
I disagree because of this being a "max length" and not a word count but I don't think it's a big deal so I've updated it.